cmd/eksctl-anywhere/cmd/generatehardware.go

```diff
@@ -91,75 +53,22 @@ func (hOpts *hardwareOptions) generateHardware(cmd *cobra.Command, args []string
 		return err
 	}
 
-	jsonDir, err := hardware.CreateDefaultJSONDir(outputDir)
-	if err != nil {
-		return err
-	}
-
 	hardwareYAML, err := os.Create(filepath.Join(outputDir, hardware.DefaultHardwareManifestYAMLFilename))
 	if err != nil {
 		return fmt.Errorf("tinkerbell manifest yaml: %v", err)
 	}
 	yamlWriter := hardware.NewTinkerbellManifestYAML(hardwareYAML)
```

Comment:
I thought the intermediate hardware.yaml file was going away. Why are we still writing to DefaultHardwareManifestYAMLFilename file?

Reply:
Its not going away. We said we'd maintain it so users can generate hardware and apply it independently.